DESCRIPTION

```diff
@@ -13,17 +13,20 @@ Suggests:
     covr,
     here,
     readr,
-    testthat (>= 3.0.0)
+    testthat (>= 3.0.0),
+    tidyxl
 Config/testthat/edition: 3
 Config/Needs/website: rmi-pacta/pacta.pkgdown.rmitemplate
 Imports: 
     dplyr,
     glue,
     janitor,
     pacta.data.validation,
+    purrr,
     rlang,
     stringr,
     tidyr,
+    unpivotr,
```

Comment:
NB: What is this package?

Reply:
this is the counterpart of {tidyxl}... {tidyxl} does the importing from XLSX files, {unpivotr} does the processing/reshpaing/etc. of tidyxl-style data frames

Reply:
yup makes sense, i put that together after reviewing the later parts haha